lib/inspec/resources/npm.rb

```diff
@@ -32,7 +32,7 @@ def info
         npm = "npm -g"
       end
 
-      cmd = inspec.command("#{npm} ls --json #{@package_name}")
+      cmd = inspec.command("sh -c '#{npm} ls --json #{@package_name}'")
```

Comment:
We should probably remove 'windows' as a supported platform.

Reply:
@miah I do not think that removing support for Windows is the way to go here.

Is it not be possible to detect the platform that Inspec is running on and choose if `sh -c` should be used for Linux or `cmd /c` for Windows?

Reply:
Thank you, on further reflection what I want to express is "there has never been correct support for windows if you use a custom path", but by removing it here we'd remove it for all users, custom path or no. 

I agree, a custom Windows-specific invocation is needed.

Reply:
This is now in place and ready for review.